tests/testing/LocalBearTestHelperTest.py

```diff
@@ -78,19 +78,11 @@ def test_bad_assertComparableObjectsEqual(self):
                                     file=fname)],
                                    filename=fname,
                                    create_tempfile=False)
-        self.assertEqual('\'LineCountTestBear\' != \'LineCountTestBea\'\n'
-                         '- LineCountTestBear\n'
-                         '?                 -\n'
-                         '+ LineCountTestBea\n'
-                         ' : origin mismatch.\n\n'
-                         '\'This file has 3 lines.\' != \'This file has 2 '
-                         'lines.\'\n'
-                         '- This file has 3 lines.\n'
-                         '?               ^\n'
-                         '+ This file has 2 lines.\n'
-                         '?               ^\n'
-                         ' : message_base mismatch.\n\n',
-                         str(cm.exception))
+        self.assertEqual('origin mismatch: LineCountTestBear, This file has '
+                         '3 lines. != LineCountTestBea, This file has 2 '
+                         'lines.\n\nmessage_base mismatch: LineCountTestBear,'
```

Comment:
please put `message_base ...` on a new line, so it is slightly easier to read.

this message change looks like it is harder to understand that the previous assertion message.